### ISSUE TITLE
能否支持Win11新版右键菜单的系统分享？ Can the system sharing of the new right-click menu of Win11 be supported? Win11新增右键菜单能否支持系统共享？

### ISSUE THREAD

**Issue by PandaFiredoge** [localsend/localsend#1509]
### Is your feature request related to a problem? Please describe.

![image](https://github.com/user-attachments/assets/cb60ebac-0173-4e80-83da-393e844ab8f0)

### Describe the solution you'd like

\

**Comment by AnnualfireCHW** [comment]
增加到windows共享菜单后会更好地与windows集成。听起来是个不错的主意。

**Comment by yeyebbc** [comment]
The easiest way to do this seems to be by using a Flutter plugin called [share_plus](https://pub.dev/packages/share_plus).

**Comment by Tienisto** [comment]
No, the share_plus plugin does the inverse: Share content from app to other apps. However, we need LocalSend to accept sharing intents

**Comment by yeyebbc** [comment]
Oh. That will be much more... inconvenient to implement.
